.github/workflows/on_pull_request_lint_title.yaml

```diff
@@ -0,0 +1,20 @@
+name: Lint PR title
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+permissions:
+  pull-requests: read
+
+jobs:
+  lint:
+    name: Validate PR title
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
```

Comment:
question: with the [default configurations](https://github.com/amannn/action-semantic-pull-request?tab=readme-ov-file#configuration), is only `Type` enforced, with `Scope` and `Subject` ignored?